src/stores/ArticleStore.js

```diff
@@ -0,0 +1,48 @@
+import { EventEmitter } from 'events';
+import dispatcher from '../dispatcher/dispatcher';
+
+/**
+ * @class ArticleStore
+ * @extends {EventEmitter}
+ */
+class ArticleStore extends EventEmitter {
+  constructor() {
+    super();
+    this.articles = [];
+  }
+
+  /**
+  * @method getArticles
+  * @return {array} - returns an array of articles
+  */
+  getArticles() {
+    return this.articles;
+  }
+
+
+  /**
+  * @method handleActions
+  * @param {obj} action
+  * @return {void}
+  * Listens to actions from the dispatcher
+  * runs actions relevant to Article Store
+  * Emits a change event
+  */
+  handleActions(action) {
+    if (action.type === 'GET_ARTICLES') {
+      this.articles = action.articles;
```

Comment:
It would be better to make a shallow copy of `action.articles` here so that the articles in your store will have a different referential identity from the ones in the action.